lib/resources/Transfers.js

```diff
@@ -6,33 +6,7 @@ const stripeMethod = StripeResource.method;
 module.exports = StripeResource.extend({
   path: 'transfers',
 
-  includeBasic: [
-    'create',
-    'list',
-    'retrieve',
-    'update',
-    'setMetadata',
-    'getMetadata',
-  ],
-
-  reverse: stripeMethod({
-    method: 'POST',
-    path: '/{id}/reversals',
-    urlParams: ['id'],
-  }),
-
-  cancel: stripeMethod({
```

Comment:
are you sure `cancel` should go?

Reply:
(in the wiki, it says it's moved to `payouts.cancel()`)

Reply:
@remi-stripe You can use `/v1/payouts/{payout}/cancel` with any API version, right? If so I think removing this is the right call.